### ISSUE TITLE
JS conflicts with third-party libraries

### ISSUE THREAD

**Issue by aastashov** [theatlantic/django-nested-admin#168]
Hi! I use the [django-admin-sortable2](https://github.com/jrief/django-admin-sortable2) library as it's convenient for me and when connecting **django-nested-admin** I saw some conflict that breaks the **django-nested-admin** library's behavior.

### Example configuration
**admin.py**
```python
from adminsortable2.admin import SortableAdminMixin
from django.contrib import admin
from nested_admin.nested import NestedModelAdmin, NestedTabularInline

class ServiceInline(NestedTabularInline):
    model = Service
    sortable_field_name = 'position'

@admin.register(Group)
class GroupAdmin(SortableAdminMixin, NestedModelAdmin):
    inlines = [
        ServiceInline,
    ]
```

And when I open the editing page, I get the next error:
![image](https://user-images.githubusercontent.com/15646544/78082117-0ad1bd00-73d4-11ea-9dc4-d0ab02b0bd5f.png)

I'm not strong in JS, but I looked at how it was done in **django-admin-sortable2** and tried to wrap the code **django-nested-admin** in the same style:

**django-nested-admin**
```javascript
'use strict';

const $ = require('jquery');

if ($.ui === undefined) {
  // the code
}

$.widget("ui.djnsortable", $.ui.mouse, {
  // the code
});
```
**django-admin-sortable2**
list-sortable.js
```javascript
"use strict";

django.jQuery(function($) {
  // the code
});
```

If I change the code in your library, it works. But it doesn't look like a ready-made solution :)).
```javascript
"use strict";

django.jQuery(function($) {
  $.widget("ui.djnsortable", $.ui.mouse, {
    // the code
  });
});
```

Now, in order not to make many changes, I found another way to solve the problem, but it's a temporary solution (I don't know how it works, but it works):
```python
# code code code
@admin.register(Group)
class GroupAdmin(SortableAdminMixin, NestedModelAdmin):
    inlines = [
        ServiceInline,
    ]

    @property
    def media(self):
        media = super().media
        return MergeSafeMedia(js=media._js, css=media._css)
```

Thank you for your work on the library!

**Comment by fdintino** [comment]
django-nested-admin has its own sortable functionality. Even if there wasn't a script incompatibility like the one you posted, django-admin-sortable2 simply wouldn't work with nested inlines: the reordering logic is more complicated, since you can drag nested inlines from one parent inline to another. Is there some feature of django-admin-sortable2 that you feel is missing from django-nested-admin's implementation?

**Comment by mihalikv** [comment]
@fdintino I have similar problem, with SortableAdminMixin. 
![image](https://user-images.githubusercontent.com/10236315/81783199-4a271980-94fb-11ea-92f5-95b008598ceb.png)
I am using SortableAdminMixin in change-list to sort objects not in inlines. It actually breaks after upgrading from Django==2.1.9 to 2.2. Following fix also works for me:

```
@property
    def media(self):
        media = super().media
        return MergeSafeMedia(js=media._js, css=media._css)
```

**Comment by fdintino** [comment]
@mihalikv Can you verify whether the [fix-media-order](https://github.com/theatlantic/django-nested-admin/tree/fix-media-order) branch resolves your issue?

**Comment by fdintino** [comment]
In the absence of feedback, I'm aAssuming that this was fixed in the 3.3.2 release, which addressed several issues with the order of media assets.

If this is still a problem, feel free to reopen this issue.
